main.py

```diff
@@ -0,0 +1,364 @@
+    edited_list = []
+    for key in list(raw_dict.keys()):
+        total = total_dict[key]
+        raw = raw_dict[key]
+
+        # First 2 weeks don't have median this handles it
+        try:
+            median = raw["Median"]
+            total_median = total["total_median"]
+            median_count = total["median_count"]
+            if median_count == 0:
+                median_diff = 0
+            else:
+                median_diff = median - (total_median / median_count)
+        except KeyError:
+            median = 0
+            median_diff = 0
+
+        current_sales = (sales * raw["pop"]) / 100
+        try:
+            avg_diff = raw["avg"] - ((total["total_avg"] - raw["avg"]) / (total["count"] - 1))
+            stddev_diff = raw["stddev"] - ((total["total_stddev"] - raw["stddev"]) / (total["count"] - 1))
+            min_diff = raw["min"] - ((total["total_min"] - raw["min"]) / (total["count"] - 1))
+            max_diff = raw["max"] - ((total["total_max"] - raw["max"]) / (total["count"] - 1))
+            pop_diff = raw["pop"] - ((total["total_pop"] - raw["pop"]) / (total["count"] - 1))
+            sales_diff = current_sales - ((total["total_sales"] - current_sales) / (total["count"] - 1))
+        except ZeroDivisionError:
+            avg_diff = 0
+            stddev_diff = 0
+            min_diff = 0
+            max_diff = 0
+            pop_diff = 0
+            sales_diff = 0
+
+        edited_list.append({
+            "itemType": raw["itemType"],
+            "compatibility": raw["compatibility"],
+            "rerolled": raw["rerolled"],
+            "avg": raw["avg"],
+            "avg_diff": avg_diff,
+            "stddev": raw["stddev"],
+            "stddev_diff": stddev_diff,
+            "min": raw["min"],
+            "min_diff": min_diff,
+            "max": raw["max"],
+            "max_diff": max_diff,
+            "pop": raw["pop"],
+            "pop_diff": pop_diff,
+            "median": raw["median"],
+            "median_diff": median_diff,
+            "sales": current_sales,
+            "sale_diff": sales_diff
+        })
+
+    edited_path = f"{EDIT_PATH.format(platform)}{filename}"
+    with open(edited_path, "w") as file:
+        json.dump(edited_list, file, indent=4)
+    print(f"Saved {filename} Edited\n-----------------\n")
+
+
+def create_dict(data, mode):
+    '''Creates the useful dicts with name as key and the data as a value
+    Mode 1 for raw data
+    Mode 2 for edited data
+    Mode 3 for Total data'''
+    data_dict = {}
+    if mode == 1:
+        for d in data:
+
+            # Creates the name for the riven
+            if d["compatibility"] is None:
+                name = f"Veiled {d['itemType']}"
+            elif d["rerolled"]:
+                name = f"{d['compatibility']}_T"
+            else:
+                name = f"{d['compatibility']}_F"
+
+            # Older data doesn't have median
+            try:
+                median = d["median"]
```

Comment:
I think we can replace this bit with `d.get('median', 0)`

Reply:
Changed.